Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -15,23 +15,10 @@ public enum PlayerSpawnMethod
     }
 
     [AddComponentMenu("Network/NetworkManager")]
-    [RequireComponent(typeof(ITransport))]
     public class NetworkManager : MonoBehaviour
     {
         // transport layer
-        // -> automatically uses the first transport component. there might be
-        //    multiple in case of multiplexing, so the order matters.
-        ITransport _transport;
-        public virtual ITransport transport
-        {
-            get
-            {
-                _transport = _transport ?? GetComponent<ITransport>();
-                if (_transport == null)
-                    Debug.LogWarning("NetworkManager has no Transport component. Networking won't work without a Transport");
-                return _transport;
-            }
-        }
+        public ITransport transport;
```

Comment:
Could this be virtual?

Reply:
no,  it is a field,  fields can't be virtual.

But you can just assign it at runtime anywhere you want